record/program-anchor/Anchor.toml

```
@@ -0,0 +1,14 @@
+[features]
+seeds = false
+[programs.localnet]
+record_anchor = "FGQLqP4smSphjS1WRamE3zEezL8tCFZrsu4Wg5z2Kq4r"
+
+[registry]
+url = "https://anchor.projectserum.com"
+
+[provider]
+cluster = "localnet"
+wallet = "/home/anurag/.config/solana/id.json"
```

So that everyone can use it :smile: 
```suggestion
wallet = "~/.config/solana/id.json"
```